HyperV/scripts/export-eventlog.ps1

```diff
@@ -19,13 +21,34 @@ function dumpeventlog($path){
 
 function exporteventlog($path){
 
-	Get-Eventlog -list | ForEach-Object {
+	Get-Eventlog -list | Where-Object { $_.Entries -ne '0' } | ForEach-Object {
 		$logfilename = "eventlog_" + $_.LogDisplayName + ".txt"
 		$logfilename = $logfilename.replace(" ","_")
 		Get-EventLog -Logname $_.LogDisplayName | fl | out-file $path\$logfilename -ErrorAction SilentlyContinue
 	}
 }
 
+function exporthtmleventlog($path){
+	$css = Get-Content $eventlogcsspath -Raw
+	$js = Get-Content $eventlogjspath -Raw
+	$HTMLHeader = @"
+	<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+	<script type="text/javascript">$js</script>
+	<style type="text/css">$css</style>
+	"@
+
+	foreach ($i in (Get-EventLog -List | Where-Object { $_.Entries -ne '0' }).Log) {
+		$Report = Get-EventLog $i
+		$Report = $Report | ConvertTo-Html -Title "${i}" -Head $HTMLHeader -As Table
+		$Report = $Report | ForEach-Object {$_ -replace "<body>", '<body id="body">'}
+		$Report = $Report | ForEach-Object {$_ -replace "<table>", '<table class="sortable" id="table" cellspacing="0">'}
+		$logName = $i  + ".html"
+		$logName = $logName.replace(" ","_")
+		$bkup = Join-Path $path $logName
+		$Report = $Report | Set-Content $bkup
+	}
+}
+
 function cleareventlog(){
```

Comment:
we must run this also as first thing when we start the init process (we need just the events related to the test init and run).